packages/grid/x-data-grid-premium/src/DataGridPremium/useDataGridPremiumProps.ts

```diff
@@ -23,6 +23,9 @@ export const DATA_GRID_PREMIUM_PROPS_DEFAULT_VALUES: DataGridPremiumPropsWithDef
   aggregationFunctions: GRID_AGGREGATION_FUNCTIONS,
   aggregationRowsScope: 'filtered',
   getAggregationPosition: (groupNode) => (groupNode.depth === -1 ? 'footer' : 'inline'),
+  disableClipboardPaste: false,
```

Comment:
Shouldn't this be added when making this feature stable i.e. removing experimental flag?

Reply:
It doesn't hurt to add it now 🙂
Also, we don't have to remember about it when removing the experimental flag.

Reply:
Hmmm, point!